### DISCUSSION TITLE
GOL TV USA

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#787] [Q&A]
### Channel Name

GOL TV

### Channel ID

GolTVUSA.us

### Website

https://play.goltv.tv/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by spotanjo3** [comment]
Hello. No. I don't want ES

**Comment by Toei79** [comment]
its not pay channel?

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19751
